source/learning-wazuh/vuln-detection.rst

```diff
@@ -111,6 +111,16 @@ In the ``/var/ossec/etc/ossec.conf`` file of the Wazuh manager, scroll down to t
         <enabled>no</enabled>
         <update_interval>1h</update_interval>
       </provider>
+      <provider name="suse">
+        <enabled>no</enabled>
+        <os>15-server</os>
```

Comment:
We will finally reverse this list to sort them from older to newer versions as the rest of providers. Please, replace them by:
```xml
      <os>11-server</os>
      <os>11-desktop</os>
      <os>12-server</os>
      <os>12-desktop</os>
      <os>15-server</os>
      <os>15-desktop</os>
```